css/index.css

```diff
@@ -0,0 +1,83 @@
+body {
+  margin: 0;
+}
+
+.calendar {
+  margin: 0 auto;
+  padding: 10px;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -ms-flex-wrap: wrap;
+      flex-wrap: wrap;
+  gap: 1px;
+  max-width: 706px;
+  counter-reset: day 0;
+}
+.calendar .calendar__day {
+  width: 100px;
+  height: 100px;
+  border: solid 1px black;
+  background-color: #eee;
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
```

Comment:
Similar to the previous comment, check if vendor prefixes for `display: flex;` are necessary. Modern browsers typically do not require these prefixes.